installation failure on R 3.5.0

Unfortunately, installation of vctrs fails on my R 3.5.0 when installing the source version 0.3.1. Apparantly it has to do with a missing Altrep.h file:

``` r
> install.packages("vctrs")

Installing package into ‘H:/Apps/R_Library_nieuw’

(as ‘lib’ is unspecified)

  There is a binary version available but the source version is later:

      binary source needs_compilation

vctrs  0.2.4  0.3.1              TRUE

installing the source package ‘vctrs’

trying URL 'https://cran.rstudio.com/src/contrib/vctrs_0.3.1.tar.gz'

Content type 'application/x-gzip' length 946964 bytes (924 KB)

downloaded 924 KB

* installing *source* package 'vctrs' ...

** package 'vctrs' successfully unpacked and MD5 sums checked

** libs

*** arch - i386

C:/Rtools/mingw_32/bin/gcc  -I"C:/Program Files/R/R-35~1.0/include" -DNDEBUG          -O3 -Wall  -std=gnu99 -mtune=generic -c altrep-rle.c -o altrep-rle.o

In file included from altrep-rle.h:4:0,

                 from altrep-rle.c:2:

altrep.h:13:26: fatal error: R_ext/Altrep.h: No such file or directory

#include "R_ext/Altrep.h"

                          ^

compilation terminated.

make: *** [C:/Program Files/R/R-35~1.0/etc/i386/Makeconf:208: altrep-rle.o] Error 1

ERROR: compilation failed for package 'vctrs'
```

--- a/src/altrep.h
+++ b/src/altrep.h
@@ -3,7 +3,8 @@
 
 #include "Rversion.h"
 
-#if (R_VERSION < R_Version(3, 5, 0))
+#if (R_VERSION < R_Version(3, 5, 0)) ||                 \
+    (defined(_WIN32) && R_VERSION == R_Version(3, 5, 0))
 
 # define ALTREP(x) false
 # define ALTVEC_EXTRACT_SUBSET_PROXY(x, indx, call) NULL

